Automattic/themes
# Spearhead: fix menu wrapping bug (#2589)

This PR fixes #2589. It also addresses a noticeable regression in the mobile menu button padding. 

Before | After
------- | -------
<img width="938" alt="Screen Shot 2020-10-14 at 2 19 57 PM" src="https://user-images.githubusercontent.com/5375500/96029330-83581280-0e28-11eb-962a-910ae1e85ca5.png"> | <img width="958" alt="Screen Shot 2020-10-14 at 2 16 39 PM" src="https://user-images.githubusercontent.com/5375500/96029324-80f5b880-0e28-11eb-8570-ac69d67c85e8.png">

**To test**
- Create and activate a primary menu with many items
- Create and activate a social menu
- View the header at viewport sizes above 828px and ensure the navigation does not wrap awkwardly
